### ISSUE TITLE
The `_generate_shapefile()` function in `reVX` has a hardcoded `CRS` value

### ISSUE THREAD

**Issue by netr0m** [NREL/reVX#167]
**Bug Description**
*Note: This may or may not be a bug, depending on whether this was a design choice. Regardless, it is affecting our workflow where we had to manually set the EPSG of the `cf_profiles` H5 output from `reV` prior to processing the data with `reVX`*

After running `RPMClusters(*args, **kwargs)._cluster()`, as per the example in [reVX docs: RPMClusters](https://nrel.github.io/reVX/_autosummary/reVX.rpm.rpm_clusters.RPMClusters.html#reVX.rpm.rpm_clusters.RPMClusters), we needed to generate a shapefile.

Running the `_generate_shapefile()` function of `reVX` sets the `CRS` to a hardcoded value of [`EPSG:5070`](https://epsg.io/5070), which covers only the United States. This is done in a manner where the _metadata_ has a CRS value of `EPSG:5070`, but the actual coordinates are never converted, which may lead to inconsistencies. (using `crs="EPSG:5070"` rather than e.g. `.to_crs(epsg=5070)`)

For our dataset, we were working with a region outside the US, and as such the coordinates in the input to/output from `reV` were in the format [`EPSG:4326`](https://epsg.io/4326) (lat/lon, in the bounds `[[-180, 90], [180, 90]]`), while the `CRS` of the output from `_generate_shapefile()` was set to `EPSG:5070` (the coordinates still being _representative_ of `EPSG:4326`). When we attempted to plot the output on a map (using [`geopandas`](https://geopandas.org/) and [`contextily`](https://contextily.readthedocs.io/en/latest/index.html)), this caused issues where the plotted region was placed at an unexpected area. By manually overriding the `CRS` of the shapefile, we managed to plot it at the correct location.

Note that we had to do:
```py
import geopandas as gpd

# Load shapefile
df = gpd.read_file(<filepath>.shp)
# Override CRS
df.crs = "EPSG:4326"
```

rather than
```py
df.to_crs(epsg=4326)
```
which would convert the values as well, despite them already being in the format of `EPSG:4326`.

See [`reVX/rpm/rpm_clusters.py#L330-L331`](https://github.com/NREL/reVX/blob/26938628e004608f687a0b5eda864eb01799e8ff/reVX/rpm/rpm_clusters.py#L330-L331)

**Full Traceback**
Full traceback of any errors received, if applicable

**Code Sample**
A copy-pastable example if possible

```python
from reVX.rpm.rpm_clusters import RPMClusters
from rex import Resource
import geopandas as gpd
import contextily as ctx

PATH_TO_CF_PROFILES_H5 = "test.h5"

# Using the output from reV.Gen (cf_profiles in an H5 file)
with Resource(cf_profiles_path) as fh:
    gen_gids = fh.meta.index.values

clusters = RPMClusters(
    PATH_TO_CF_PROFILES_H5,
    gen_gids,
    n_clusters=6
)
clusters._cluster()
RPMClusters._generate_shapefile(clusters.meta, fpath="test.shp")

# Load the output shapefile from _generate_shapefile()
df = gpd.read_file("test.shp")

ax = df.plot(figsize=(10, 10), alpha=0.5, edgecolor='k')
ctx.add_basemap(ax, zoom=10, crs=df.crs)
```

**To Reproduce**
Steps to reproduce the problem behavior
1. Generate `cf_profiles` using `reV.Gen`
2. Load the `cf_profiles` using `reVX`
3. Do clustering with `RPMClusters()._cluster()`
4. Generate a shapefile with `RPMClusters._generate_shapefile()`
5. Load the shapefile with geopandas (`df = geopandas.read_file(shapefile_path)`)
6. Look at the `crs` attribute of the dataframe, `df.crs`, which is now using `EPSG:5070`, while the coordinates may not be representative (e.g. still in `EPSG:4326`, as in our case)

**Expected behavior**
That the generated shapefile uses a CRS that covers more than just the United States.

**Screenshots**
N/A

**System (please complete the following information):**
 - OS: Arch Linux, but running inside Docker (`continuumio/miniconda3` / `python v3.9`)
 - Compute hardware: N/A
 - Version: v0.3.46

**Additional context**
N/A

**Charge code**
N/A

**Comment by grantbuster** [comment]
Hey @netr0m, thanks for using our code and thanks for the detailed issue report!

I'm honestly not sure why we defaulted to EPSG:5070, I don't think i've ever used that projection before. All of our data is typically in EPSG:4326 which i think is the same as yours. That may have been a mistake. The corresponding commit doesn't say anything informative about that choice. 

In either case, can you review the following PR and let me know if you need anything extra added besides the extra crs kwargs?

https://github.com/NREL/reVX/pull/168

**Comment by netr0m** [comment]
Hey @grantbuster, thanks for the quick response! The PR looks good, I believe this will resort our issue! :1st_place_medal:

**Comment by grantbuster** [comment]
Awesome. We're figuring out a mysterious version bug in the test suite then i'll merge and close this.

**Comment by grantbuster** [comment]
@netr0m, this was implemented in #168 and released in https://github.com/NREL/reVX/releases/tag/v0.3.48

Note that I also removed the leading underscore from the `generate_shapefile()` method name since this should obviously be a public method.
